unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/FriendsHUDController.cs

```diff
@@ -243,10 +249,17 @@ private void UpdateUserStatus(string userId, UserStatus status)
                 friend.blocked = IsUserBlocked(userId);
                 friends[userId] = friend;
                 View.Set(userId, friend);
+
+                if (status.presence == PresenceStatus.ONLINE)
+                    onlineFriends[userId] = friend;
+                else
+                    onlineFriends.Remove(userId);
```

Comment:
we should also consider to remove online friends on `FriendshipStatus.NOT_FRIEND` & `FriendshipStatus.REQUESTED_TO`